src/doghouse/core/ports/storage_port.py

```
@@ -0,0 +1,21 @@
+from abc import ABC, abstractmethod
+from typing import List, Optional
```

Ruff (0.15.7)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)